discord_bot/discord_bot.py

```diff
@@ -144,24 +142,39 @@ def load_config(config_file_path):
                     self.logger.warning("Invalid Session")
                 elif message["op"] == Opcodes.DISPATCH:
                     event = message["t"]
-                    if event == Events.READY:
+                    if event == Events.READY.value:
                         self.session_id = message["d"]["session_id"]
+                    elif event == Events.MESSAGE_CREATE.value:
+                        await self.respond_message(message)
                 else:
                     self.logger.exception("Unexpected opcode {}: {}".format(message["op"], message))
 
-    # TODO: Remove this test code
-    # async def send_message(self, recipient_id, content):
-    #     """
-    #     Post a message into the given channel
-    #     :param recipient_id: num - the recipient to open a DM channel with
-    #     :param content: obj - message sent
-    #     :return: dict - Fires a "Message Create" Gateway event
-    #     """
-    #     channel = requests.post("{}", json={"recipient_id": recipient_id}).json()
-    #
-    #     return requests.post(
-    #         "{}/channels/{}/messages".format(
-    #             self.config["discord_api_endpoint"], channel["id"]),
-    #         json={
-    #             "content": content
-    #         }).json()
+    async def respond_message(self, message):
+        """
+        Receives a message, parses the message, and responds to the user in the same channel.
+        :param message: obj - the payload of the MESSAGE_CREATE event
+        :return: string - defined by data
+        """
+        channel_id = message["d"]["channel_id"]
+        author_id = message["d"]["author"]["id"]
+        content = message["d"]["content"]
+        if author_id != self.config["momoi_id"]:
+            if content.startswith("!help"):
+                async with ClientSession() as session:
+                    async with session.request(
+                            "POST",
+                            "{}/channels/{}/messages".format(self.config["discord_api_endpoint"], channel_id),
```

Comment:
You can do this in a separate PR, but we should probably convert all of our `"blah{}".format(myVar)` to `f"blah{myVar}"` for consistency